db/migrate/20151024080849_collections_belongs_to_many_projects.rb

```diff
@@ -0,0 +1,8 @@
+class CollectionsBelongsToManyProjects < ActiveRecord::Migration
+  def change
+    add_column :collections, :project_ids, :integer, array: true, default: []
+    Collection.where.not(project_id: nil).update_all("project_ids = ARRAY[project_id]")
+    add_index  :collections, :project_ids, using: 'gin'
```

Comment:
Weren't there issues with this? I thought we didn't have this index on `classifications.subject_ids` for some specific reason?